### PR TITLE
Reflect workload tally across various modules

### LINKED ISSUE #89: Reflect workload tally across the different modules

### PR DESCRIPTION
**Pull request by yhtMinceraft1010X**
Fixes #89

### CODE REVIEW

File: src/main/java/seedu/module/model/task/Module.java [hunk 1]
```diff
@@ -60,6 +61,27 @@ public static boolean isSupportedModuleCode(String test) {
         return existingModules.contains(test);
     }
 
+    public int getWorkloadCount() {
+        return workloadCount;
+    }
```
Comment (umergta) [inline src/main/java/seedu/module/model/task/Module.java]: hmm correct me if im wrong, but youll be adding the raw workload of each task to the module? Maybe, as a better indicator for modules, we can calculate the proportion / fraction of the total workload for each module? I think that wll be a more useful number for the user? Maybe the rest can pitch in here too

**Review by umergta — COMMENTED** [review]
generally LGTM other than the logic bit (ive commented there) but it may require a change in the current implementation. Solid efficiency, Yi Hseun!

File: src/main/java/seedu/module/model/task/Module.java (hunk 1 above)
Comment (yhtMinceraft1010X) [reply src/main/java/seedu/module/model/task/Module.java]: I've changed it to show how many tasks there are across all 3 workload ratings for a single module.

File: src/main/java/seedu/module/model/task/Module.java [hunk 2]
```diff
@@ -61,10 +65,57 @@ public static boolean isSupportedModuleCode(String test) {
...
+        return String.format("Low workload tasks: %d, medium workload tasks: %d, high workload tasks %d\n",
+                lowWorkloadCount, mediumWorkloadCount, highWorkloadCount);
+    }
+
+    /**
+     * Increases count of number of tasks in the relevant workload rating.
      */
+    public void incrementWorkload(Workload workload) {
+        requireNonNull(workload);
+        assert !(workload.getWorkloadLevel() < 1 || workload.getWorkloadLevel() > 3);
```
Comment (figo2127) [inline src/main/java/seedu/module/model/task/Module.java:80]: nice defensive programming

File: src/main/java/seedu/module/model/task/Module.java [hunk 3]
```diff
@@ -61,10 +65,57 @@ public static boolean isSupportedModuleCode(String test) {
     }
 
     /**
-     * Returns a list of existing Modules
-     *
-     * @return List of existing Modules
+     * Returns a count of the number of tasks of each workload rating.
+     */
+    public String getWorkloadCount() {
+        return String.format("Low workload tasks: %d, medium workload tasks: %d, high workload tasks %d\n",
```
Comment (figo2127) [inline src/main/java/seedu/module/model/task/Module.java:71]: maybe medium and high's first letter should be capitalized like Low?

**Review by figo2127 — COMMENTED** [review]
Overall looks good! Just need some tweaks

File: src/main/java/seedu/module/model/task/Module.java (hunk 1 above)
Comment (figo2127) [reply src/main/java/seedu/module/model/task/Module.java]: cool! Wld be good too if u can include a total workload count for each module? Thought that'd be a nice thing to have.

Comment (yhtMinceraft1010X) [reply src/main/java/seedu/module/model/task/Module.java]: Just to clarify, it's just re-add the original workload count implementation? 

So we have:
-One variable counting how many low workload tasks there are
-One for medium workload tasks
-One for high workload tasks
-One that sums up the workload rating translated to int

**Review by QY-H00 — COMMENTED** [review]
LGTM.

**Review by AiwassPrime — COMMENTED** [review]
LGTM

**Pull request merged**